src/comparable.cr

```diff
@@ -51,4 +70,13 @@ module Comparable(T)
   # [4, 7, 2].sort { |x, y| y <=> x } # => [7, 4, 2]
   # ```
   abstract def <=>(other : T)
+
+  private def _compare_with(other : T)
```

Comment:
Why does this method name begin with an underscore? This is not typically used for private methods.

Reply:
I'm afraid that if someone includes `Comparable` and wants to define a `compare_with` method (for some other purpose) it will conflict with this method.

There are no real private methods in Crystal, everything leaks.

Reply:
Okay, sounds legit. But what about putting the code directly into each method? A little bit duplication is not an issue.  Then there would simply be no method that could be accidentally overridden.
And it would get rid of the somewhat ugly block short cuts `&.>(0)`.

Reply:
Good idea. Done!